data/data-pipeline/data_pipeline/score/score_l.py

```diff
@@ -93,6 +94,22 @@ def _combine_island_areas_with_states_and_set_thresholds(
 
         return df, threshold_column_name
 
+    def create_low_income_threshold(
```

Comment:
Nit: let's make this `_create_low_income_threshold` to indicate that it's a private method that shouldn't be used outside of the class. https://stackoverflow.com/questions/1301346/what-is-the-meaning-of-single-and-double-underscore-before-an-object-name#:~:text=Python%20doesn't%20have%20real,not%20part%20of%20the%20API.

Reply:
Otherwise looks great!! Nice docstrings 😄